src/course-unit/data/api.js

```diff
@@ -52,27 +52,31 @@ export async function editUnitDisplayName(unitId, displayName) {
 }
 
 /**
- * Get sequence metadata for a given sequence ID.
- * @param {string} sequenceId - The ID of the sequence for which metadata is requested.
- * @returns {Promise<Object>} - A Promise that resolves to the normalized sequence metadata.
+ * Get an object containing course section vertical data.
+ * @param {string} unitId
+ * @returns {Promise<Object>}
  */
-export async function getSequenceMetadata(sequenceId) {
+export async function getCourseSectionVerticalData(unitId) {
   const { data } = await getAuthenticatedHttpClient()
-    .get(getSequenceMetadataApiUrl(sequenceId), {});
+    .get(getCourseSectionVerticalApiUrl(unitId));
 
-  return normalizeSequenceMetadata(data);
+  return normalizeCourseSectionVerticalData(data);
 }
 
 /**
- * Get an object containing course section vertical data.
- * @param {string} unitId
- * @returns {Promise<Object>}
+ * Sends a new sequence navigation unit.
+ * @param {string} sequenceId - The ID of the parent sequence.
+ * @returns {Promise<object>} A Promise that resolves with the server response data.
  */
-export async function getCourseSectionVerticalData(unitId) {
+export async function sendNewSequenceNavigationUnit(sequenceId) {
```

Comment:
[important] Also it looks like a duplicated functional https://github.com/raccoongang/frontend-app-course-authoring/blob/ts-develop/src/course-unit/data/api.js#L117

Reply:
Yes you are right 👍 
A lot has changed during the life of this pull request.